scipy/interpolate/interpnd.pyx

```diff
@@ -832,6 +864,8 @@ class CloughTocher2DInterpolator(NDInterpolatorBase):
         Rescale points to unit cube before performing interpolation.
         This is useful if some of the input dimensions have
         incommensurable units and differ by many orders of magnitude.
+    xi : tuple of ndarrays, optional
+        The coordinates of the points to be interpolated.
```

Comment:
This is not a parameter of the `__init__` function.

Reply:
Removed.